### PR TITLE
[new release] ke (v0.3)

### PR DESCRIPTION
**Pull request by dinosaure**
Queue implementation

- Project page: <a href="https://github.com/mirage/ke">https://github.com/mirage/ke</a>
- Documentation: <a href="https://mirage.github.io/ke/">https://mirage.github.io/ke/</a>

##### CHANGES:

* Add `Rke.{,Weighted}.compress` function (fuzzed)
* Update `bechamel` benchmark
* Add `Rke{,.Weighted}.N.peek` function
* Fix bug on `Rke.Weighted.N.keep` function
* Add some tests

### CODE REVIEW

File: packages/ke/ke.v0.3/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+homepage:     "https://github.com/mirage/ke"
+bug-reports:  "https://github.com/mirage/ke/issues"
+dev-repo:     "git+https://github.com/mirage/ke.git"
+doc:          "https://mirage.github.io/ke/"
+license:      "MIT"
+synopsis:     "Queue implementation"
+description:  """Queue implementation in OCaml (functional and imperative queue)"""
+
+build: [
+  [ "dune" "subst" ]
```
Comment (kit-ty-kate) [inline packages/ke/ke.v0.3/opam]: do you really need to call `dune subst` everytime ?

**Comment by kit-ty-kate** [comment]
Looks good to merge anyway. Thanks!

**Comment by hannesm** [comment]
curious whether you intended to release `ke.0.3` instead of `ke.v0.3` (i.e. exclude the `v` from the version number in opam) -- would imho be good to `git mv packages/ke/ke.v0.3 packages/ke/ke.0.3` //cc @dinosaure

**Comment by dinosaure** [comment]
Yes, I did not know that `dune-release` considers the `v` now in front of the version in the `CHANGES.md` - because, I did not change anything about my way to write the `CHANGES,md` ... I agree with your request. I will make a PR soon - however, we need to know why `dune-release` has a new behavior now.

**Comment by hannesm** [comment]
@dinosaure for me, dune-release with changes such as in [tcpip](https://raw.githubusercontent.com/mirage/mirage-tcpip/master/CHANGES.md) works fine (releases are prefixed with `v` in https://github.com/mirage/mirage-tcpip/releases, the PR generated uses tcpip.3.7.4).

**Comment by mseri** [comment]
@kit-ty-kate It has just been released, we could do a fast move to 0.3 before anybody uses it

**Comment by dinosaure** [comment]
@hannesm I believe, next time I need to take care about this or update `CHANGES.md` to remove `v` prefix - the second solution seems good.

**Comment by hannesm** [comment]
@dinosaure you are using in ke
```
v0.3
-----
```

maybe this is treat different from `### v0.3` (as I'm usually doing)

**Comment by dinosaure** [comment]
@hannesm seems fix the problem with the latest version of `dune-release` (which generate a proper `ke.0.3` with `dune-release opam pkg` - it seems related to samoht/dune-release#5).

**Pull request merged**